com.playeveryware.eos/Runtime/Core/PlatformManager.cs

```diff
@@ -209,28 +214,20 @@ public static Type GetConfigType(Platform platform)
             return PlatformInformation[platform].ConfigType;
         }
 
-#endif
-        /// <summary>
-        /// Return a string that represents the file extension used by the indicated platform for dynamic library files.
-        /// </summary>
-        /// <param name="platform">The platform to get the specific file extension for.</param>
-        /// <returns>File extension for the dynamic library used by the indicated platform.</returns>
-        public static string GetDynamicLibraryExtension(Platform platform)
+        public static bool TryGetConfigType(Platform platform, out Type configType)
         {
-            return PlatformInformation[platform].DynamicLibraryExtension;
-        }
+            configType = null;
 
+            bool typeFound = PlatformInformation.TryGetValue(platform, out PlatformInfo value);
```

Comment:
I feel like `PlatformInformation.TryGetValue` should return false if a null value is detected.

Reply:
Again, in this circumstance there is an `Unknown` platform, for which `PlatformInfo` should be null - but we may want to define some characteristics of it within the `PlatformInfo` struct.